libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetworkTests/MockNetworkTests.m

```diff
@@ -126,4 +130,13 @@ - (void)dont_testNetworkContexts {
     XCTAssertFalse(action5.cancelled);
 }
 
+// Test that the cached CSFNetwork is removed on logout
+- (void) testLogout {
+    SFUserAccount *user = [TestDataAction testUserAccount];
```

Comment:
That's the new test, get a CSFNetwork, check cache of CSFNetwork's, logout, check cache of CSFNetwork's again.